lib/libzfs/libzfs_changelist.c

```diff
@@ -553,39 +553,44 @@ change_one(zfs_handle_t *zhp, void *data)
 	return (0);
 }
 
-/*ARGSUSED*/
 static int
-compare_mountpoints(const void *a, const void *b, void *unused)
+compare_props(const void *a, const void *b, zfs_prop_t prop)
 {
 	const prop_changenode_t *ca = a;
 	const prop_changenode_t *cb = b;
 
-	char mounta[MAXPATHLEN];
-	char mountb[MAXPATHLEN];
+	char propa[MAXPATHLEN];
+	char propb[MAXPATHLEN];
 
-	boolean_t hasmounta, hasmountb;
+	boolean_t haspropa, haspropb;
 
-	/*
-	 * When unsharing or unmounting filesystems, we need to do it in
```

Comment:
Can we put this comment somewhere else instead of removing it?
Maybe in the body of `compare_mountpoints`?